docs/content/latest/explore/ysql-language-features/queries.md

```diff
@@ -910,68 +921,41 @@ CREATE TABLE employees (
   manager_no integer,
   department text
 );
+```
 
-INSERT INTO employees (employee_no, name, manager_no, department) 
-  VALUES 	
-  (1221, 'John Smith', NULL, NULL),
-  (1222, 'Bette Davis', 1221, 'Sales'),
-  (1223, 'Lucille Ball', 1221, 'Operations'),
-  (1224, 'John Zimmerman', 1222, 'Sales'), 
-  (1225, 'Walter Marx', 1222, 'Sales');
+```sql
+INSERT INTO employees VALUES 
+(1221, 'John Smith', NULL, NULL),
```

Comment:
why delete the leading spaces (here and lots of other places)? they work fine on my machine, and make it easier to read the multi-line statements...

Reply:
Added spaces here, but I've had so many problems with them that I'd rather keep examples less readable. It could be related to the user's browser. e.g, I use Firefox